TheWarWithin/RogueAssassination.lua

```diff
@@ -531,6 +532,10 @@ spec:RegisterHook( "spend", function( amt, resource )
     end
 end )
 
+spec:RegisterStateExpr( "poisonChance", function ()
```

Comment:
Generally, "expressions" should use SimC's token format since they could theoretically be used in a priority.  `poison_chance`

Reply:
Would it be better to rewrite it as a `local poisonChance = function () return foo end` in that case, somewhere above the ability registrations?